sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -446,6 +475,7 @@ export interface CloudEvent<T> {
     extensionAttributes?: Record<string, unknown>;
     id: string;
     source: string;
+    specversion?: string | "1.0";
```

Comment:
any chance to name this `specVersion`?

Reply:
But it is actually specversion in tsp file